Source/Port/Reference-Impl/mbedOS/RTOS/pal_plat_rtos.cpp

```diff
@@ -21,54 +21,26 @@
 #include "stdlib.h"
 #include "string.h"
 
-#include "cmsis_os.h" // Revision:    V1.02
-// These includes try to find declaration of NVIC_SystemReset.
-// Note: At least on A9 the cmsis_nvic.h can not be included without previous
-// board specific includes, so it has to be included only when known to work.
-#if defined (__CORTEX_M0)
-#include "cmsis_nvic.h"
-#include "core_cm0.h"
-#elif defined (__CORTEX_M3)
-#include "cmsis_nvic.h"
-#include "core_cm3.h"
-#elif defined (__CORTEX_M4)
-#include "cmsis_nvic.h"
-#include "core_cm4.h"
-#elif defined (__CORTEX_M7)
-#include "cmsis_nvic.h"
-#include "core_cm7.h"
-#else
-// Declaration of NVIC_SystemReset is not present on core_ca9.h, but in the
-// board specific files. 
-void NVIC_SystemReset(void);
-#endif
-
-#include "critical.h"
+#include "mbed.h"
```

Comment:
We don't need that compatibility - if someone wants to use the older one, they will have to keep the mbed-os.lib and pal.lib in sync. Anyone using mbed OS 5.5 or newer - use this, older ones use the PAL r1.1 from pal.lib (i.e. hash before this one).

The mbed-os-example-client keeps these in sync, so if you check out any version of the client -> you will get a working combination.

@yogpan01